src/routes/governance/vote-details.scss

```diff
@@ -0,0 +1,88 @@
+@import "../../styles/colors";
+
+.proposal-toast {
```

Comment:
Why have we got a proposal-toast class? We don't have any toasts in this project